src/scenes/home/Task.js

```diff
@@ -1,27 +1,35 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import {
-  View,
+  TouchableOpacity,
   Text,
 } from 'react-native';
 
 import styles from './Task.styles';
 
 export default function Task(props) {
-  const { title, description } = props;
+  const {
+    id,
+    title,
+    description,
+    onPress,
+  } = props;
+
   return (
-    <View style={styles.item}>
+    <TouchableOpacity style={styles.item} onPress={() => onPress(id)}>
```

Comment:
You should avoid defining lambda functions inside components. What about defining this function like an external function?

_**export const action = (id, onPress) => () => onPress(id);**_

_**<TouchableOpacity style={styles.item} onPress={action(id, onPress)}>**_